src/Data/Fin/Permutation/Transposition/List.agda

```diff
@@ -0,0 +1,71 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Decomposition of permutations into a list of transpositions.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Fin.Permutation.Transposition.List where
+
+open import Data.Fin.Base
+open import Data.Fin.Permutation
+import Data.Fin.Permutation.Components as PC
+import Data.List as L
+open import Data.Nat.Base using (ℕ; suc; zero)
+open import Data.Product using (_×_; _,_)
+open import Function using (_∘_)
+open import Relation.Binary.PropositionalEquality as P
+  using (_≡_; _≢_; refl; trans; sym; →-to-⟶; cong; cong₂)
+open P.≡-Reasoning
+
+
+------------------------------------------------------------------------
+-- Types
+
+TranspositionList : ℕ → Set
+TranspositionList n = L.List (Fin n × Fin n)
+
+------------------------------------------------------------------------
+-- Operations on transposition lists
+
+evalₜ : ∀ {n} → TranspositionList n → Permutation′ n
```

Comment:
The nice thing about the new file is that we can now just call these `eval` and `lift₀`